src/useClientRect/useClientRect.ts

```diff
@@ -3,7 +3,7 @@ import { useState, useCallback } from 'react'
 export function useClientRect () {
```

Comment:
Maybe define a type `Rect` in this file that you can use instead retyping `ClientRect | DOMRect | null` everywhere

Reply:
I agree, can I change this or maybe better @Michiel87 give his opinion as well